### ISSUE TITLE
Menu options present in the 'Tool strip container' dialog are not accessible using keyboard keys: A11y_.NET CoreWinforms_ToolStripContainer_Toptool bar_Keyboard

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/winforms#11451]
### .NET version

dotnet-sdk-9.0.100-preview.5.24258.1

### Did it work in .NET Framework?

Yes

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #BM_.NET Core[Winforms]_Win32_May2024; #.NETCore; #WCAG2.1.1; #DesktopApp; #Win11; #FTP; #Keyboard;

### Environment Details:
App name: .NET Core Winforms
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "MenuToolbars_Controls:Testing the controls under Menu Toolbars Tab" and hit ENTER key.
4. TAB to ToolStripContainer button and hit ENTER key.
5. TAB to top toolbar section and observe the issue.

### Actual Result:
Menu options 'Menustrip, File, Edit, Tools and Help" present in the 'Tool strip container' dialog are not accessible using keyboard keys.

### Expected Result:
Menu options 'Menustrip, File, Edit, Tools and Help" present in the 'Tool strip container' dialog should be accessible using keyboard keys.

### User Impact:
Keyboard users will face difficulty while interacting with the controls and will not be able to perform any action related to it.

### Attachment

https://github.com/dotnet/winforms/assets/90901519/90c87997-c0aa-452f-a390-1bfc244dbf73

**Comment by merriemcgaw** [comment]
@PoojaNamde this is something that developers can set in their own apps very easily.
@LeafShi1 can your team or @Olina-Zhang's please update the sample app. We can then provide a new version of the sample app that has all the fixes in it.

**Comment by LeafShi1** [comment]
@PoojaNamde The test app has been updated, please download the latest version.

**Comment by LeafShi1** [comment]
The current problem page consists of ToolStribcontainer1.ContentPanel and toolStripContainer1.TopToolStripPanel. We only need to set ToolStripContainer1.toptoolStrippanel.Tabstop = True

**Comment by Nora-Zhou01** [comment]
As shown below, after selecting Menustrip, you can use Alt+left/right arrow to move the focus to the last or next one to access the menu options "Menubar, File, Edit, Tools and Help" in the "Toolstrip Container" dialog box.
Controls with label attributes cannot be accessed with the keyboard.

https://github.com/dotnet/winforms/assets/104609169/fe60c509-1d37-4b64-a3c6-f5dc43f329cb

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;

**Comment by PoojaNamde** [comment]
@LeafShi1 Could you please share the latest environment to reverify this issue.

**Comment by Olina-Zhang** [comment]
@PoojaNamde The test app and SDK have been updated, please download the updated app and .NET 9.0 SDK to test. It should be as following:

https://github.com/dotnet/winforms/assets/26474449/e2d5a1b1-e400-4001-b471-c31f2808d077

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:06-05-24;

Verified the bug in latest environment
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment


https://github.com/dotnet/winforms/assets/90901519/9ef3f2d1-13fa-47b8-b582-c4f3e1a25989
